sqlmesh/core/plan/definition.py

```diff
@@ -255,6 +255,11 @@ def to_evaluatable(self) -> EvaluatablePlan:
             models_to_backfill=self.models_to_backfill,
             interval_end_per_model=self.interval_end_per_model,
             execution_time=self.execution_time,
+            disabled_restatement_snapshots={
```

Comment:
[Nit] It looks like the convention is to use "models" every time we reference by name and use "snapshot" every time we reference by SnapshotId / Snapshot. Let's make it `models`.